src/library/librarytablemodel.cpp

```diff
@@ -101,3 +105,13 @@ TrackModel::CapabilitiesFlags LibraryTableModel::getCapabilities() const {
             | TRACKMODELCAPS_CLEAR_BEATS
             | TRACKMODELCAPS_RESETPLAYED;
 }
+
+void LibraryTableModel::setSort(int column, Qt::SortOrder order)
+{
+    BaseSqlTableModel::setSort(column, order);
+
+    // Random sort easter egg, only in library view
+    if (column == 1) {
```

Comment:
Use
fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_PREVIEW)
to make the code robust against future changes.